packages/core/quri_parts/core/estimator/__init__.py

```diff
@@ -185,6 +185,24 @@ def parametric_estimator(
     return parametric_estimator
 
 
+def create_concurrent_parametric_estimator(
+    parametric_estimator: ParametricQuantumEstimator[_ParametricStateT],
+) -> ConcurrentParametricQuantumEstimator[_ParametricStateT]:
+    """Creates concurrent parametric estimator from parametric estimator."""
+
+    def concurrent_parametric_estimator(
+        operator: Estimatable,
+        state: _ParametricStateT,
+        seq_of_params: Sequence[Sequence[float]],
+    ) -> list[Estimate[complex]]:
+        ret = []
+        for params in seq_of_params:
+            ret += [parametric_estimator(operator, state, params)]
+        return ret
```

Comment:
I'm a bit worried about efficiency of list concatenation.
Using list comprehension might be better?

Reply:
Thank you for your comment!
I've replaced that part with a list comprehension.